finals-2024-01-16/code/4660-darko-ivanovski/src/app/components/band-edit/band-edit.component.html

```diff
@@ -0,0 +1,22 @@
+<div *ngIf="band">
```

Comment:
No form around the form elements - i.e. the submit button does not know what to submit. +2 points for the component